src/store/store.ts

```diff
@@ -0,0 +1,14 @@
+import { createStore } from "redux";
+
+let intialState = {
+  user: {},
+};
+function formReducer(state = intialState, action:any) {
+  state={...state}
+  if (action.type === "ADD_DATA") {
+    state.user=action.payload;
+  }
```

Comment:
Use Switch case here instead of if and use Types for "ADD_DATA" as you can see in stackStream code , also follow the directory order please